src/components/NameListContext.js

```diff
@@ -29,14 +29,24 @@ class NameListProvider extends React.Component {
       preferences: [],
       sortBy: "name",
       namesPerPage: 25,
-
+      chartData: [],
+      filterDisplay: "visible",
+      // Functional Export
       onChangeOrder: this.onChangeOrder,
       onChangeNumber: this.onChangeNumber,
       onAdd: this.onAdd,
       onDelete: this.onDelete,
       callList: this.callList,
+      callChart: this.callChart,
+      toggleDisplay: this.toggleDisplay,
     };
   }
+  // Toggle the filter form modal
+  toggleDisplay = () => {
```

Comment:
minor suggestion for deduplicating code and making this easier to read:
```javascript
const filterDisplay = this.state.filterDisplay === "visible" ? "hidden" : "visible";
this.setState({ filterDisplay })
```

Reply:
Good Catch. Thanks